doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown

```diff
@@ -163,7 +161,7 @@ or viewed instantly with `xdot` command (please refer to your
 distribution/operating system documentation on how to install these
 packages).
 
-![Anisotropic image segmentation graph](pics/segm.gif)
+![Anisotropic image segmentation graph](pics/segm.png)
```

Comment:
> segm.gif

Is file still used? If not, remove it too.

Reply:
removed